buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -36,14 +36,14 @@ object Dependencies {
     const val guava = "com.google.guava:guava:30.1.1-android"
     const val squareup_okhttp = "com.squareup.okhttp3:okhttp:${Versions.okhttp3}"
     const val squareup_okhttp_tls = "com.squareup.okhttp3:okhttp-tls:${Versions.okhttp3}"
-    const val burgstaller_okhttp_digest = "com.burgstaller:okhttp-digest:2.5"
+    const val burgstaller_okhttp_digest = "io.github.rburgst:okhttp-digest:2.5"
     const val persian_joda_time = "com.github.mohamadian:persianjodatime:1.2"
     const val myanmar_calendar = "com.github.chanmratekoko:myanmar-calendar:1.0.6.RC3"
     const val bikram_sambat = "bikramsambat:bikram-sambat:1.1.0"
     const val danlew_android_joda = "net.danlew:android.joda:2.10.12.2"
-    const val rarepebble_colorpicker = "com.rarepebble:colorpicker:3.0.1"
+    const val rarepebble_colorpicker = "com.github.martin-stone:hsv-alpha-color-picker-android:3.0.1"
     const val commons_io = "commons-io:commons-io:2.5" // Commons 2.6+ introduce java.nio usage that we can't access until our minSdkVersion >= 26 (https://developer.android.com/reference/java/io/File#toPath())
-    const val opencsv = "net.sf.opencsv:opencsv:2.4"
+    const val opencsv = "net.sf.opencsv:opencsv:2.3"
```

Comment:
We should make sure to highlight the effected areas (just the CSV secondary instances) for regression testing. This dependency is old and unmaintained so needs to go anyway (I'll file a separate issue).

Reply:
We should use Apache Commons CSV to match JavaRosa

Reply:
Yeah, I think we might even have that pulled in to our classpath anyway. To be clear, I'm not going to switch that over as part of this PR.